.github/workflows/cli.yml

```diff
@@ -125,17 +125,30 @@ jobs:
           touch ~/.grafbase/version.txt
         shell: bash
 
+      - name: install binstall
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: 'cargo-binstall'
+
+      - name: install nextest
+        uses: actions-rs/cargo@v1
+        with:
+          command: binstall
+          args: 'cargo-nextest --secure'
+
       - name: test
         uses: actions-rs/cargo@v1
         with:
-          command: test
+          command: nextest
+          args: '--test-threads 1'
```

Comment:
What's the reason for that? Doesn't it defeat one of the main advantages of using nextest?

Reply:
The CLI currently can't be run more than once in the exact same time due to finding a port not being an atomic operation (two CLIs find the same bridge port and one of them errors), so it can't run in parallel yet, will look into it soon. I tried creating a struct which holds a listener and a port number to be able to drop it before using the port (effectively "locking" it), but that has problems with the OS freeing the port in time and probably isn't the most efficient

Reply:
Ah, in that case you might want to add a [config file](https://github.com/grafbase/api/blob/main/.config/nextest.toml) (with a comment) so that nextest will work locally and in CI.